kubernetes/utils/create_from_yaml.py

```diff
@@ -257,6 +257,10 @@ def create_from_yaml_single_item(
         kwargs.pop('namespace', None)
         resp = getattr(k8s_api, "create_{0}".format(kind))(
             body=yml_object, **kwargs)
+
+    if kwargs.get('async_req'):
```

Comment:
If `async_req=True` is specified, isn't it the callers responsibility to examine the result? 
Here we are turning an async request to sync, is it what the caller would expect? 
cc @ChrisOwenBoring

Reply:
It will require more changes then, I believe (i.e. handling `if verbose:` down below, callers return type and mention that they won't raise `FailToCreateError` for `async_req`)